eq_translations/survey_schema.py

```diff
@@ -137,6 +157,26 @@ def get_answer_pointers(self):
         """
         return find_pointers_to(self.schema, "label")
 
+    def get_placeholder_pointers(self):
+        """
+        Placeholder 'text' may require context for translation
+        """
+        (
+            context_placeholder_pointers,
+            no_context_placeholder_pointers,
+        ) = self._get_pointers_for_key("text")
+        return context_placeholder_pointers, no_context_placeholder_pointers
+
+    def get_plural_pointers(self):
+        """
+        'text_plural' may require context for translation
+        """
+        (
+            context_plural_pointers,
+            no_context_plural_pointers,
+        ) = self._get_pointers_for_key("text_plural")
+        return context_plural_pointers, no_context_plural_pointers
```

Comment:
I don't see why these two methods exist, couldn't `self._get_pointers_for_key("text|text_plural")` just be called in place of them?

Reply:
This is what I did first then changed it because it's what we do with:
- find_pointers_to(self.schema, "title")
- find_pointers_to(self.schema, "label")

So I changed it to be similar, but more than happy to change back.